README.md

```diff
@@ -171,47 +171,48 @@ can easily configure them for you specific needs.
 <td align="center">DS18B20</td>
 </tr><tr>
 <td align="center">EA-DOG</td>
+<td align="center">ENCODER-SLAVE</td>
```

Comment:
Should be `ENCODER-OUTPUT` (maybe also something more descriptive like AB Encoder Output)